app/models/pursuit.rb

```diff
@@ -3,9 +3,9 @@ class Pursuit < ActiveRecord::Base
   has_many :ordered_trips
   has_many :orders, through: :ordered_trips
 
-  validates :name, presence: true
-  validates :description, presence: true
-  validates :price, presence: true
+  validates :name, :description, :price, :activity_id, presence: true
+  validates_numericality_of :price, greater_than: 0
```

Comment:
Prefer the new style validations `validates :column, numericality: value` over `validates_numericality_of`.